src/Illuminate/Console/Scheduling/CallbackEvent.php

```diff
@@ -50,47 +65,53 @@ public function __construct(EventMutex $mutex, $callback, array $parameters = []
     }
 
     /**
-     * Run the given event.
+     * @return void
+     *
+     * @throws \RuntimeException
+     */
+    public function runInBackground()
+    {
+        throw new RuntimeException('Callbacks can not be scheduled to run in the background.');
+    }
+
+    /**
+     * Run the callback event.
      *
      * @param  \Illuminate\Contracts\Container\Container  $container
      * @return mixed
      *
-     * @throws \Exception
+     * @throws \Throwable
      */
     public function run(Container $container)
     {
-        if ($this->description && $this->withoutOverlapping &&
-            ! $this->mutex->create($this)) {
-            return;
-        }
+        parent::run($container);
```

Comment:
If we don't want to throw a `RuntimeException` when `runInBackground()` is called, we could do this instead:

```suggestion
        $this->runInBackground = false;
        parent::run($container);
```

I don't love it, because it implies that the callback will be run in the background even though it won't (and currently doesn't in 8.x).